log/development.log

```diff
@@ -16806,6 +16806,511 @@ Started GET "/assets/users-fcec5b5a277ac7c20cc9f45a209a3bcd.js?body=1" for ::1 a
 Started GET "/assets/application-eebb8f1e18a153cec2f8ad7e1a90d685.js?body=1" for ::1 at 2015-03-25 20:34:01 -0400
```

Comment:
These log file shouldn't be in git.  please make sure to remove them and make sure they don't get added again.